eslint.config.mjs

```diff
@@ -38,6 +38,9 @@ export default [
         },
         "rules": {
             "semi": ["error", "always"],
+            "vue/html-self-closing": [
+                "error", { "html": { "void": "any" } }
```

Comment:
If we don't like the `<div>` elements being touched, there should be a way to turn this off with `normal: any`